Sources/SnapshotTesting/Snapshotting/SwiftUIView.swift

```diff
@@ -37,6 +37,14 @@ extension Snapshotting where Value: SwiftUI.View, Format == UIImage {
     traits: UITraitCollection = .init()
     )
     -> Snapshotting {
+      #if compiler(>=5.7)
+      if #available(iOS 16, *), drawHierarchyInKeyWindow == false {
```

Comment:
Should this have the `tvOS` check as seen below, as well? Any other platforms?

Reply:
Should definitely add `tvOS` check

We're inside of a `#if os(iOS) || os(tvOS)` check here, so those are the only relevant OSs

Reply:
@john-flanagan I meant shouldn't the `#available` check include `tvOS 16`?